src/Business/Services/AdditionalFacilityService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using AutoMapper;
+using Business.Exceptions;
+using Business.Interfaces;
+using Business.Models;
+using Data;
+using Data.Entities;
+using Data.Interfaces;
+
+namespace Business.Services
+{
+    public class AdditionalFacilityService : IAdditionalFacilityService
+    {
+        private readonly ApplicationContext _context;
+        private readonly IAdditionalFacilityRepository _additionalFacilityRepository;
+        private readonly IMapper _mapper;
+
+        public AdditionalFacilityService(IMapper mapper, ApplicationContext context, IAdditionalFacilityRepository additionalFacilityRepository)
+        {
+            _mapper = mapper;
+            _context = context;
+            _additionalFacilityRepository = additionalFacilityRepository;
+        }
+
+        public async Task<IList<AdditionalFacilityModel>> GetListAsync()
+        {
+            var entities = await _additionalFacilityRepository.GetListAsync();
+
+            return _mapper.Map<IList<AdditionalFacilityEntity>, IList<AdditionalFacilityModel>>(entities);
```

Comment:
why are you using IList? I guess, that IEnumerable will be enough

Reply:
Understood.
Should i return IEnumerable from business and data layer, or only from business?